ansible/roles/ci/code_style_check/python/tasks/main.yml

```
@@ -23,7 +23,7 @@
          
     source /opt/ros/{{ros_release}}/setup.bash && rosrun roslint test_wrapper \
     {{ros_workspace}}/build/test_results/{{item}}/roslint-python3-ex-{{item}}.xml \
-    "pylint -sn --rcfile=/tmp/pylintrc --msg-template='{abspath}:{line}:{column}:{msg_id}:{msg}' {{item}} & sort --field-separator=':' -k 2,2n"
+    "pylint -sn --rcfile=/tmp/pylintrc --msg-template='{abspath}:{line}:{column}:{msg_id}:{msg}' {{item}}"
```

What is the ampersand supposed to do? Did you mean to pipe?
```suggestion
    "pylint -sn --rcfile=/tmp/pylintrc --msg-template='{abspath}:{line}:{column}:{msg_id}:{msg}' {{item}} | sort --field-separator=':' -k 2,2n"
```